sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -43,7 +45,7 @@ class DatalabAppElement extends Polymer.Element {
   constructor() {
     super();
 
-    // set the pattern once to be the current document pathname
+    // Sets the pattern once to be the current document pathname
```

Comment:
No need to use the third-person version form ("Sets") here, as it is not a function comment. One of the reasons to do so on functions is to avoid any abiguity that the comment might be a command to the caller of the function.